core/trino-main/src/main/java/io/trino/server/security/KerberosAuthenticator.java

```diff
@@ -204,7 +203,7 @@ T get()
 
     private static <T> T doAs(Subject subject, GssSupplier<T> action)
```

Comment:
nit: This is used only in `KerberosAuthenticator` can we use `callAs` here also ?